src/constants/externalResources.ts

```diff
@@ -3,56 +3,66 @@ import { environmentVariables } from "utils";
 
 const { getLobsterURL, getParsleyUrl, getUiUrl } = environmentVariables;
 
-const wikiBaseUrl = "https://docs.devprod.prod.corp.mongodb.com/evergreen";
+export const wikiBaseUrl =
+  "https://docs.devprod.prod.corp.mongodb.com/evergreen";
 
-const wikiUrl = `${wikiBaseUrl}/Home`;
+export const wikiUrl = `${wikiBaseUrl}/Home`;
 
-const projectDistroSettingsDocumentationUrl = `${wikiBaseUrl}/Configure%20a%20Project/Project-and-Distro-Settings`;
+export const projectDistroSettingsDocumentationUrl = `${wikiBaseUrl}/Configure%20a%20Project/Project-and-Distro-Settings`;
 
-const versionControlDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#version-control`;
+export const versionControlDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#version-control`;
 
-const patchAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#patch-aliases`;
+export const patchAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#patch-aliases`;
 
-const pullRequestAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#pr-aliases`;
+export const pullRequestAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#pr-aliases`;
 
-const commitQueueAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#commit-queue-aliases`;
+export const commitQueueAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#commit-queue-aliases`;
 
-const gitTagAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#git-tag-aliases`;
+export const gitTagAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#git-tag-aliases`;
 
-const githubChecksAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#github-checks-aliases`;
+export const githubChecksAliasesDocumentationUrl = `${projectDistroSettingsDocumentationUrl}#github-checks-aliases`;
 
-const cliDocumentationUrl = `${wikiBaseUrl}/Using-the-Command-Line-Tool`;
+export const cliDocumentationUrl = `${wikiBaseUrl}/Using-the-Command-Line-Tool`;
 
-const windowsPasswordRulesURL =
+export const windowsPasswordRulesURL =
   "https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc786468(v=ws.10)?redirectedfrom=MSDN";
 
-const konamiSoundTrackUrl =
+export const jiraBugUrl =
+  "https://jira.mongodb.org/secure/CreateIssueDetails!init.jspa?pid=12787&issuetype=1&priority=4&labels=user-feedback&description=Please%20note%20browser%20and%20OS%20when%20describing%20your%20issue.";
+
+export const jiraImprovementUrl =
+  "https://jira.mongodb.org/secure/CreateIssueDetails!init.jspa?pid=12787&issuetype=4&priority=4&labels=user-feedback";
```

Comment:
The SpruceConfig stores the `jira.host`, which is maybe what we'd want to use here? I think that's what the `getJiraSearchUrl` and `getJiraTicketUrl` functions do